releases/latest/k8s/kafka-k8s/README.md

```diff
@@ -0,0 +1,14 @@
+# Kafka + Zookeeper Kubernetes Bundle
```

Comment:
Should this be a link to the [kafka-k8s readme](https://github.com/canonical/kafka-k8s-operator/blob/main/README.md) instead? Is there a specific way to handle bundle docs?

Reply:
I believe that @marcoppenheimer wrote that README.  Idc either way, but I believe that it is more specific to the bundle than the operator